docs/pt/docs/tutorial/path-params-numeric-validations.md

```diff
@@ -0,0 +1,122 @@
+# Parâmetros da Rota e Validações Numéricas
+
+Do mesmo modo que você pode declarar mais validações e metadados para parâmetros de consulta com `Query`, você pode declarar os mesmos tipos de validações e metadados para parâmetros de rota com `Path`.
+
+## Importe `Path`
+
+Primeiro, importe `Path` de `fastapi`:
+
+```Python hl_lines="3"
+{!../../../docs_src/path_params_numeric_validations/tutorial001.py!}
+```
+
+## Declare metadados
+
+Você pode declarar todos os parâmetros da mesma maneira que na `Query`.
+
+Por exemplo para declarar um valor de metadado `title` para o parâmetro de rota `item_id` você pode digitar:
+
+```Python hl_lines="10"
+{!../../../docs_src/path_params_numeric_validations/tutorial001.py!}
+```
+
+!!! nota
+    Um parâmetro de rota é sempre obrigatório como se fosse parte da rota.
+    
+    Logo, você deve declará-lo com `...` para marcá-lo como obrigatório.
+
+    Mesmo que você declare-o como `None` ou defina um valor padrão, isso não teria efeito algum, o parâmetro ainda seria obrigatório.
+
+## Ordene os parâmetros de a acordo com sua necessidade
+
+Suponha que você quer declarar o parâmetro de consulta `q` como uma `str` obrigatória.
```

Comment:
```suggestion
Suponha que você declare o parâmetro de consulta `q` como uma `str` obrigatória.
```

Reply:
In this case specifically I think your suggestion has a slightly change in the original idea of a person thinking in declare the query param.

Reply:
what do you think of "Suponha que você queira declarar o parâmetro de consulta `q` como uma `str` obrigatória"?

Reply:
That's nice! I created a new commit with this suggestion, thank you!